ONSdigital/sdx-transform-cs
# Fix construction yes/no question transformation

## What? and Why?
There are 4 questions with yes/no answers that weren't being transformed correctly.  This PR makes them 1 or 2 depending on yes/no.

## How to test
- spin up console
- Put the following json into the submission (or you can generate your own by going to the eq go launcher).  This Json has 2 of the questions (reporting dates and housing work) as yes and the other two (around infrastructure and other stuff) as no.
```
{"case_id":"2d993570-f899-460a-b0dc-b3f6e34aacb7","collection":{"exercise_sid":"0bb6d77d-8934-416f-aef4-c2e32246edb8","instrument_id":"0001","period":"201605"},"data":{"146":"comment","201":"9000","202":"1000","211":"3000","212":"1000","290":"14000","901":"Yes, I can report for this period","902":"Yes, we carried out work on housing","903":"No, we did not carry out work on infrastructure","904":"No, we did not carry out other construction work","d1":"['Public sector projects', 'Private sector projects']"},"flushed":false,"metadata":{"ref_period_end_date":"2016-05-31","ref_period_start_date":"2016-05-01","ru_ref":"28358254342e","user_id":"UNKNOWN"},"origin":"uk.gov.ons.edc.eq","started_at":"2020-04-16T14:28:58.198252","submitted_at":"2020-04-16T14:29:44.800328","survey_id":"228","tx_id":"56d7eb49-22d1-45a0-91a2-e7fb7298d12b","type":"uk.gov.ons.edc.eq:surveyresponse","version":"0.0.1"}
```

Check the PCK has 901 and 902 (the qcodes reporting dates and housing) have 1 and 903 and 904 have 2.  Also check the image to make sure it looks reasonable.

## Checklist
